examples/adsb/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "adsb_demod"
+version = "0.1.0"
+edition = "2021"
+
+[workspace]
+
+[[bin]]
+name = "listen-adsb"
+path = "src/bin/listen_adsb.rs"
+
+[dependencies]
+clap = { version = "4.0.27", features = ["derive"] }
+futuresdr = { path = "../..", features=["soapy", "audio"] }
+futuredsp = { path = "../../futuredsp" }
+futuresdr-pmt = { path = "../../pmt" }
+serde = { version = "1.0", features = ["derive"] }
+serde_with = "2.1.0"
+serde_json = "1.0.89"
+env_logger = "0.10.0"
+adsb_deku = { git = "https://github.com/rsadsb/adsb_deku" }
```

Comment:
I pushed an updated library that builds, should be able to change this to `adsb_deku = "0.6.2"`.

Tested this myself and it worked fine.

Reply:
Thanks! It works for me as well.